script/QA/QA_17.txt

```diff
@@ -26,72 +26,78 @@ A merciless, spicy monster.
 [sha:31b7215b11b365c8551cc9de2af78eef9cf951d6]{
 -- Page 101, Offset 43502.
 -- 主人公におかれましては
-As for the protagonist,
+As for the protagonist, just give%{n}
```

Comment:
"the protagonist" could probably use a bit more spice
"as for Mr. Protagonist" or something, otherwise it sounds kinda weirdly impersonal.